# --- T2-COPYRIGHT-BEGIN ---
# t2/package/*/libcap/hotfix-psx_calls.patch.ia64
# Copyright (C) 2025 The T2 SDE Project
# SPDX-License-Identifier: GPL-2.0 or patched project license
# --- T2-COPYRIGHT-END ---

diff --git a/psx/psx_calls.c b/psx/psx_calls.c
index 99e1d51..807cc2a 100644
--- a/psx/psx_calls.c
+++ b/psx/psx_calls.c
@@ -33,7 +33,7 @@
  * want to use, we have ended up just inlining all architecture
  * support here.
  */
-#if defined(__x86_64__) || defined(__i386__)	   \
+#if defined(__x86_64__) || defined(__i386__) || defined(__ia64__) \
     || defined(__arm__) || defined(__aarch64__)	   \
     || defined(__mips__) || defined(__loongarch__) \
     || defined(__powerpc__) || defined(__s390__) || defined(__riscv)
